ccp/src/main/java/com/hbb20/Country.java

```diff
@@ -871,6 +873,7 @@ static int getFlagResID(Country country) {
         countries.add(new Country("hk", "852", "Hong Kong"));
         countries.add(new Country("hu", "36", "Hungary"));
         countries.add(new Country("in", "91", "India"));
+        countries.add(new Country("ic", "354", "Iceland"));
```

Comment:
This list is only used for preview purpose. On runtime, list is picked from xml files.